app/src/main/java/com/example/scoutconcordia/ShuttleInfo.java

```diff
@@ -0,0 +1,542 @@
+                        nextShuttleTime = shuttleTimes[i];
+                        indexPosition = i;
+                        break;
+                    }
+                }
+
+                messageToUser = "The current time is: " + timeString + " (" + timeOfDay + ")" + " and the next bus shuttle comes at: " + schedule_MonToThurs_SGW[indexPosition] + " (" + nextShuttleTime + ")";
+
+                break;
+
+            case "Wednesday":
+                shuttleTimes = getSGWMondayToThursdayTimes();
+
+                for (int i = 0; i < shuttleTimes.length; i++){
+                    if (timeOfDay < shuttleTimes[i]){
+                        nextShuttleTime = shuttleTimes[i];
+                        indexPosition = i;
+                        break;
+                    }
+                }
+
+                messageToUser = "The current time is: " + timeString + " (" + timeOfDay + ")" + " and the next bus shuttle comes at: " + schedule_MonToThurs_SGW[indexPosition] + " (" + nextShuttleTime + ")";
+
+                break;
+
+            case "Thursday":
+                shuttleTimes = getSGWMondayToThursdayTimes();
+
+                for (int i = 0; i < shuttleTimes.length; i++){
+                    if (timeOfDay < shuttleTimes[i]){
+                        nextShuttleTime = shuttleTimes[i];
+                        indexPosition = i;
+                        break;
+                    }
+                }
+
+                messageToUser = "The current time is: " + timeString + " (" + timeOfDay + ")" + " and the next bus shuttle comes at: " + schedule_MonToThurs_SGW[indexPosition] + " (" + nextShuttleTime + ")";
+
+                break;
+
+            case "Friday":
+
+                shuttleTimes = getSGWFridayTimes();
+
+                for (int i = 0; i < shuttleTimes.length; i++){
+                    if (timeOfDay < shuttleTimes[i]){
+                        nextShuttleTime = shuttleTimes[i];
+                        indexPosition = i;
+                        break;
+                    }
+                }
+
+                messageToUser = "Today is " + today + ". The current time is: " + timeString + " (" + timeOfDay + ")" + " and the next bus shuttle comes at: " + schedule_friday_SGW[indexPosition] + " (" + nextShuttleTime + ")";
+
+                break;
+
+
+            case "Saturday":
+                messageToUser = "There are no bus shuttles on Saturdays!";
+                break;
+        }
+
+        // The shuttle time is hardcoded at 30 minutes, because of fluctuations, it is an estimation
+        double estimatedTime =  Math.round((((nextShuttleTime - timeOfDay)*60 ) + 30) * 100.0) / 100.0;
+
+        String overallEstimation = "The estimated route time is: " + estimatedTime;
+
+        System.out.println((messageToUser + "\n" + overallEstimation + " minutes"));
+
+//            return (messageToUser + "\n" + overallEstimation);
+
+        return estimatedTime;
+
+    }
```

Comment:
This function seems to be quite long. Is there any way to refactor this so that it's shorter and easier to read?

Reply:
Same thing with getNextEarliestTimeFromLoyola().

Reply:
> Same thing with getNextEarliestTimeFromLoyola().

Ofcourse, as soon as we know for sure that we won't adding method-specific changes (such as specific hard-coded traffic times), I'll refactor into smaller methods. For now, I just wanted to make sure it works.

Reply:
I believe taking into account traffic is beyond the scope of this project. We can safely assume a consistent transit time between campuses.

Maybe @JenYn can weigh in on this.

Reply:
> I believe taking into account traffic is beyond the scope of this project. We can safely assume a consistent transit time between campuses.
> 
> Maybe @JenYn can weigh in on this.

There were talks of increasing the hard-coded "traffic time" during peak hours, however I do agree with you that it is beyond the scope of what we want to do.

Reply:
After consideration, I agree that hard-coding "traffic time" is not necessary and beyond the scope of the course. The additional time will not be accurate if hard-coded.

Reply:
@vicmanea Now that it's settled that we won't be implementing traffic or variable changes, can you refactor these long functions into shorter ones? If you have any issues, let us know.